src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -828,6 +828,10 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile(
                                             application->mac_address());
   }
 
+  if (resumption) {
```

Comment:
I think there should be an additional condition to check that result_code != RESUME_FAILED. Alternative approach would be to set resumption to false in the event of RESUME_FAILED result code set above.

Reply:
@JackLivio Thank you. Fixed in d38383d186ecab67a35a7acd90aeaf8a94832b93

Reply:
@JackLivio FYI. PR was rebased to the lates develop as new features were merged. All physical and logical issues were fixed. Unit tests fixed/updated. ATF scripts were rebased as well.